mkosi/distributions/gentoo.py

```diff
@@ -35,40 +33,39 @@ def invoke_emerge(
             "--usepkg=y",
             "--keep-going=y",
             "--jobs",
-            "--load-average",
+            # Make sure the system stays somewhat usable by putting the max load average at 90%.
+            f"--load-average={0.9 * len(os.sched_getaffinity(0))}",
             "--nospinner",
             "--root-deps=rdeps",
             "--with-bdeps=n",
             "--complete-graph-if-new-use=y",
             "--verbose-conflicts",
             "--changed-use",
             "--newuse",
-            f"--root={Path('/tmp/mkosi-root')}",
+            "--root=/tmp/mkosi-root",
+            "--binpkg-respect-use",
             *(["--verbose", "--quiet=n", "--quiet-fail=n"] if ARG_DEBUG.get() else ["--quiet-build", "--quiet"]),
             *options,
-            *actions,
         ],
         bwrap_params=[
             "--bind", state.root, "/tmp/mkosi-root",
             "--bind", state.cache_dir / "binpkgs", "/var/cache/binpkgs",
             "--bind", state.cache_dir / "distfiles", "/var/cache/distfiles",
-            "--bind", state.cache_dir / "repos", "/var/db/repos",
+            "--bind", state.cache_dir / "repos", "/var/db/repos/gentoo",
```

Comment:
this would disable [overlays](https://overlays.gentoo.org/). i would keep the mnt-point at `/var/db/repo`